src/ipaddr.coffee

```diff
@@ -387,6 +387,15 @@ ipaddr.parseCIDR = (string) ->
     catch e
       throw new Error "ipaddr: the address has neither IPv6 nor IPv4 CIDR format"
 
+ipaddr.parseBinary = (bytes) ->
+  length = bytes.length
+  if length == 4
+    return new ipaddr.IPv4(bytes)
+  else if length == 8
+    return new ipaddr.IPv6(bytes)
```

Comment:
Adding support for parsing a 16x8 array to `IPv6.parse` makes a lot of sense, yeah